apps/11gist-blog/script.js

```diff
@@ -1,5 +1,58 @@
 'use strict';
 
-$(document).ready(function() {
-  // You code here
+$(document).ready(function () {
+  $.ajax('https://api.github.com/users/JamesPuckett/gists', { //http://127.0.0.1:8080/apps/11gist-blog/api/gists.json
+    success: function (response) {
+      response.forEach(function (post) {
+        var $description = post.description;
+        var $url = post.url;
+        var $comments = post.comments_url;
+        //only uses posts that start with '#post ' and removes the string '#post '
+        //creates a link that has to data attritubutes so i can reference them later with ajax calls
+        if ($description.substring(0, 6) === '#post ') {
+          var $str = `
+          <li>
+            <a href='#' data-comments=${$comments} data-url=${$url}>${$description.replace('#post ', '')}
+            </a>
+          </li>
+          `;
+          $('#posts').append($str);
+        }
+        //this section below here does not appear to be working.
```

Comment:
Looks like the issue you're having is that the title of the file is expected to be 'post.md' but yours is coming back as gistfile1.txt